specification/indices/_types/IndexSettings.ts

```diff
@@ -311,8 +312,8 @@ export class IndexSettingsAnalysis {
 }
 
 export class IndexSettingsTimeSeries {
-  end_time?: DateOrEpochMillis
-  start_time?: DateOrEpochMillis
+  end_time?: DateTime
```

Comment:
This property type should still be a union, maybe `DateTime | EpochMillis`?

Reply:
`DateTime` is defined as `string | long` (actually will be `string | EpochTime<UnitMillis>` in the next commit) as ES accepts both a formatted date (generally ISO format) and an epoch-millis as input for `DateTime` even if it always outputs it as an ISO-formatted date, except, sometimes, in the settings.

The reason why in this specific case it can be a number is because settings are stored verbatim as expressed in the JSON that defines them. So if this setting was defined as an epoch millis number, it will be sent back as that same number.